server/services/return-of-service.js

```diff
@@ -112,8 +128,12 @@ const updateReturnOfServiceStatus = async ({ participantId, newSite, newDate, ne
     throw new Error(rosError.fieldNotFound);
   }
 
-  if (newSite && !rosParticipant.site_id) {
-    throw new Error(rosError.noSiteAttached);
+  let newSiteId;
+  if (newSite) {
+    if (!rosParticipant.site_id) {
+      throw new Error(rosError.noSiteAttached);
+    }
+    newSiteId = await getRosSiteId(newSite);
```

Comment:
What's the need for the database ID here? It seems like an extra database call just to switch from one ID to the other, is the id in `newSite` not appropriate for logging?

Reply:
So, the `employer_sites` table has 2 ids: `id` and `siteId`; I need to link site to the `ros` table, so I need to know which id to use
`newSite` = `siteId` (I cannot attach it as FK to ros)
`newSiteId` = `id`, which I'm using as an actual siteId in this case to create relation

<img width="750" alt="ScreenShot" src="https://user-images.githubusercontent.com/64768811/184735849-cc74aee0-ab70-4a2d-aa0b-3b3c45a657ba.png">